workflows/src/test/kotlin/com/r3/corda/lib/tokens/workflows/factories/TestEvolvableTokenTypeFactory.kt

```diff
@@ -21,31 +21,31 @@ class TestEvolvableTokenTypeFactory(val testSuite: JITMockNetworkTests, val main
         )
     }
 
-    fun withOneMaintainerAndOneObserver(linearId: UniqueIdentifier = UniqueIdentifier(), maintainer: Party = testSuite.identity(maintainerA), observer: Party = testSuite.identity(observerC)): TestEvolvableTokenType {
+    fun withOneMaintainerAndOneParticipant(linearId: UniqueIdentifier = UniqueIdentifier(), maintainer: Party = testSuite.identity(maintainerA), observer: Party = testSuite.identity(observerC)): TestEvolvableTokenType {
```

Comment:
How come the rename ? The function still has an `observer` parameter

Reply:
It's because it wasn't an observer in the oberverSessions sense. it is additional participant on evolvable token type. The definition of `TestEvolvableTokenType` uses observers only to add them to participants list.